### DISCUSSION TITLE
Upgrade team error (Failed to upgrade team. Please try again.)

### DISCUSSION THREAD

**Discussion by Vlad202** [vercel/community#3563] [Account Issue] [answered]
### Steps to Reproduce

Hello. We are trying to pay for the pro version of our vercel, but the payment is not going through. We tried for several days with different bank cards, but we get an error. We have $0.50 in our account for a test payment, but the funds are still not debited and no information is sent to the bank.

**Comment by amyegan** [comment, marked as answer]
Sorry you're having account upgrade issues, @Vlad202. Due to the sensitive nature of account and billing information, we cannot dig into those details in this public discussion. Please [submit a case](https://vercel.com/help#issues) to the Customer Success Team with `Billing` as the problem area.
